pkgs/applications/science/misc/golly/default.nix

```
@@ -35,22 +36,37 @@ stdenv.mkDerivation rec {
     wrapGAppsHook3
   ];
 
+  enableParallelBuilding = true;
+
+  preConfigure = lib.optionalString stdenv.isDarwin ''
+    export NIX_CFLAGS_COMPILE+=" -L$SDKROOT/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries"
+  '';
+
   setSourceRoot = ''
     sourceRoot=$(echo */gui-wx)
   '';
 
-  postPatch = ''
-    sed -e 's@PYTHON_SHLIB@${python3}/lib/libpython3.so@' -i wxprefs.cpp
-    ! grep _SHLIB *.cpp
-
-    grep /lib/libpython wxprefs.cpp
-  '';
+  postPatch =
+    ''
+      sed -e 's@PYTHON_SHLIB@${python3}/lib/libpython3.so@' -i wxprefs.cpp
```

```suggestion
      substituteInPlace wxprefs.cpp \
        --replace-fail 'PYTHON_SHLIB' '${python3}/lib/libpython3.so'
```
Since you removed the grep invocations that checked for successful sed usage.